coil-gif/src/main/java/coil/decode/GifDecoder.kt

```diff
@@ -54,6 +58,21 @@ class GifDecoder : Decoder {
 
         drawable.setRepeatCount(options.parameters.repeatCount() ?: MovieDrawable.REPEAT_INFINITE)
 
+        // Set the start and end animation callbacks if any one is supplied through the request.
+        if (options.parameters.animationStartCallback() != null || options.parameters.animationEndCallback() != null) {
+            drawable.registerAnimationCallback(object : Animatable2Compat.AnimationCallback() {
+                override fun onAnimationStart(drawable: Drawable?) {
+                    super.onAnimationStart(drawable)
```

Comment:
The `super` implementations for `Animatable2` and `Animatable2Compat` are all no-ops so we don't need to call them.

Reply:
Noted